.gitignore

```diff
@@ -8,6 +8,7 @@ pnpm-debug.log*
 lerna-debug.log*
 package-lock.json
 yarn.lock
+pnpm-lock.yaml
```

Comment:
**Issue:** we are using pnpm so ignoring `pnpm-lock.yaml` will cause our CI to break.

Reply:
I wouldn't add `pnpm-lock` at all as legacy should be installed via `npm` - see github workflows as example: https://github.com/saleor/macaw-ui/blob/canary/.github/workflows/main.yml#L20-L22